helpers/qespresso/dati_mc.generate_crystal

```diff
@@ -36,7 +36,7 @@ flag_disk=T                                      !Save all the data on files for
 flag_output=T                                    !Save all the outputs on the output.d file
 quick_error=4		                         !Number of blocks used to estimate the error with the blocking technique. If quick_error=0 then the optimal value is determined automatically
 flag_random_file=T                               !Use a file with true random numbers to initialize the pseudo-random number generator
-random_seed_path='RANDOM_SEED_FOLDER/randomseed1.d'    !Path to the file used to initialized the pseudo-random number generator
+random_seed_path='randomseed1.d'    !Path to the file used to initialized the pseudo-random number generator
```

Comment:
The problem is, that my i-pi client expects a randomseed file within the working directory, so that the file can safely be changed by seedfile_shift.py. Therefore I decided to let pilot's set_dir copy a seedfile by default and changed the input templates appropriately.